src/cli/commands/bitswap/unwant.js

```diff
@@ -1,11 +1,21 @@
 'use strict'
 
+const print = require('../../utils').print
+
 module.exports = {
   command: 'unwant <key>',
 
-  describe: 'Remove a given block from your wantlist.',
+  describe: 'Removes a given block from your wantlist.',
 
+  builder: {
+    key: {
+      alias: 'k',
+      describe: 'Key to remove from your wantlist',
+      type: 'string'
+    }
+  },
   handler (argv) {
-    throw new Error('Not implemented yet')
+    argv.ipfs.bitswap.unwant(argv.key);
```

Comment:
From the other cli files I looked at that use callbacks, if they get an error they just throw an error. I don't think the cli has a "catch and log to stderr" functionality in place.

I was using `src/cli/commands/bitswap/wantlist.js` and `src/cli/commands/block/get.js` as reference points on this because one is from the bitswap command list and the other is the "opposite" of unwant.

Reply:
Yes, you're totally right that we don't have a pattern for handling failures on CLI. That's probably issue-able, I'll add it to a todo. My thought was that we at least shouldn't `print('Key.. removed')` after a failure.

Reply:
As far as I can tell all failures will throw an exception so the print won't happen.  Under the hood it's just removing an item from a set, once the CID validation passes (which would throw if there was an invalid key passed).